src/main/java/com/microsoft/azure/functions/worker/handler/WorkerStatusRequestHandler.java

```diff
@@ -0,0 +1,17 @@
+package com.microsoft.azure.functions.worker.handler;
+import com.microsoft.azure.functions.rpc.messages.*;
+
+public class WorkerStatusRequestHandler extends MessageHandler<WorkerStatusRequest, WorkerStatusResponse.Builder> {
+
+    public WorkerStatusRequestHandler() {
+        super(StreamingMessage::getWorkerStatusRequest,
+              WorkerStatusResponse::newBuilder,
+              null,
+              StreamingMessage.Builder::setWorkerStatusResponse);
+    }
+
+    @Override
+    String execute(WorkerStatusRequest request, WorkerStatusResponse.Builder response) {
```

Comment:
Please correct me if I am wrong, this message will not matter as the WorkerStatusResponse is empty in the protobuf message.

Reply:
True, but just leaving it there for future too when we actually update it with contents.